examples/keras_io/generative/gpt2_text_generation_with_kerasnlp.py

```diff
@@ -0,0 +1,375 @@
+"""
+Title: GPT2 Text Generation with KerasNLP
+Author: Chen Qian
+Date created: 04/17/2023
+Last modified: 04/17/2023
+Description: Use KerasNLP GPT2 model and `samplers` to do text generation.
+Accelerator: GPU
+"""
+
+"""
+In this tutorial, you will learn to use [KerasNLP](https://keras.io/keras_nlp/) to load a
+pre-trained Large Language Model (LLM) - [GPT-2 model](https://openai.com/research/better-language-models)
+(originally invented by OpenAI), finetune it to a specific text style, and
+generate text based on users' input (also known as prompt). You will also learn
+how GPT2 adapts quickly to non-English languages, such as Chinese.
+"""
+
+"""
+##  Before we begin
+
+Colab offers different kinds of runtimes. Make sure to go to **Runtime ->
+Change runtime type** and choose the GPU Hardware Accelerator runtime
+(which should have >12G host RAM and ~15G GPU RAM) since you will finetune the
+GPT-2 model. Running this tutorial on CPU runtime will take hours.
+"""
+
+"""
+## Install KerasNLP, Choose Backend and Import Dependencies
+
+This examples uses [Keras 3.0](https://keras.io/keras_core/) to work in any of
+`"tensorflow"`, `"jax"` or `"torch"`. Support for Keras Core is baked into
+KerasNLP, simply change the `"KERAS_BACKEND"` environment variable to select
+the backend of your choice. We select the JAX backend below.
+"""
+
+"""shell
+pip install git+https://github.com/keras-team/keras-nlp.git -q
+"""
+
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"  # or "tensorflow" or "torch"
```

Comment:
You can remove these 2 lines if it works with any backend.

Reply:
done